lib/DataSift/ApiClient.php

```diff
@@ -67,7 +67,12 @@ static public function call(
         $ssl = $user->useSSL();
 
         // Build the full endpoint URL
-        $url = 'http'.($ssl ? 's' : '').'://'.$user->getApiUrl(). $user->getApiVersion() . '/'. $endPoint;
+        if ($method == 'ingest') {
```

Comment:
@datasift-chrisknight You're right it's not exactly nice but it was an easy solution to this given the current structure of the library. Happy to talk it through if you have an idea on how to handle multipel URLs